API-Mocks/WordPressMocks/src/main/assets/mocks/mappings/files.json

```diff
@@ -0,0 +1,13 @@
+{
+    "request": {
+        "method": "GET",
+        "urlPattern": "/.*([A-z0-9]*\\.[A-z]{1,6})"
```

Comment:
This regex matches request for files in multiple folder levels `.*`, file names with numbers, upper and lower case letters `[A-z0-9]*` and extensions up to 6 letters long `\\.[A-z]{1,6}`.